### DISCUSSION TITLE
Library installation problem?

### DISCUSSION THREAD

**Discussion by AammarTufail** [AammarTufail/python_ka_chilla2023#16] [Q&A]
Please comment here

**Comment by Atifidrees2023** [comment]
![photo_2023-01-06_02-19-25](https://user-images.githubusercontent.com/121822088/210852754-4a530c5c-41db-41ba-8730-18b3f11ecbe5.jpg)

**Reply by AammarTufail** [reply]
vscode band kar k new file se start karen and aaj ki video dobara dekhen

**Comment by Atifidrees2023** [comment]
![image](https://user-images.githubusercontent.com/121822088/210852832-7f13f36a-a094-4b11-8952-0255d0146f8d.png)

**Reply by Masoodullah909** [reply]
Restart your VSCode or Restart your Kernel.

**Comment by mhamza6051** [comment]
all libraries install but cannot install polty library
![image](https://user-images.githubusercontent.com/121826823/210853869-a6ea5108-4066-4552-aafd-0d01cb980a3d.png)

**Reply by baberibrar** [reply]
> all libraries install but cannot install polty library ![image](https://user-images.githubusercontent.com/121826823/210853869-a6ea5108-4066-4552-aafd-0d01cb980a3d.png)

It's **plotly** not ploty

**Reply by mhamza6051** [reply]
solution b batta ditty tu acha tha is ka

**Reply by SahilJatoi744** [reply]
bhaijan apki requirements satisfied hai ap neche text parhe to ap bar bar kyu libraries install kar rahe hai

**Reply by Muhammad-Ali-Butt** [reply]
> solution b batta ditty tu acha tha is ka

@baberibrar ne solution hi bataya bhai pip install ploty likha howa aapnay usko pip install **plotly** karo ge tu install ho ga plotly
